src/go/k8s/api/redpanda/v1alpha2/redpanda_types.go

```diff
@@ -40,6 +40,22 @@ type ChartRef struct {
 	Timeout *metav1.Duration `json:"timeout,omitempty"`
 	// Defines how to handle upgrades, including failures.
 	Upgrade *HelmUpgrade `json:"upgrade,omitempty"`
+	// NOTE! Alpha feature
+	// UseFlux flag set to `false` will prevent helm controller from reconciling helm chart. The operator would be
+	// tight with `go` based Redpanda helm chart version. The rest of the ChartRef fields would be ignored.
+	//
+	// Before setting UseFlux flag to `false` please alight your ChartVersion to at least `5.9.3`
```

Comment:
Bit of an odd request, when you pull `5.9.3` into a constant. Could you add a test that asserts that the generated CRD references the constant so we can be sure it doesn't get out of date? Similar to what is being done here: https://github.com/redpanda-data/helm-charts/blob/main/pkg/lint/lint_test.go

Reply:
Done